source/LocalizationManager.cs

```diff
@@ -113,13 +119,14 @@ private static Localizer CreateLocalizer (Localizer fallback, string culture, IT
 				}
 				catch (Exception e)
 				{
-					throw new TemplateSetLoadException(culture, setname, e);
+					throw new TemplateSetLoadException(culture, "SOMETHING", e); //TODO This is WRONG
```

Comment:
An exception with `setname` information needs to be thrown from the provider, because the manager no longer knows about setnames